dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProcessDefinitionMapper.xml

```diff
@@ -139,4 +139,12 @@
         set version = #{version}
         where id = #{processDefinitionId}
     </update>
+
+    <select id="queryDefinitionByIdAndVersion" resultType="org.apache.dolphinscheduler.dao.entity.ProcessDefinition">
+        select
+        <include refid="baseSql"/>
```

Comment:
You only need to know if there is data, so it is recommended to return the ID. At the same time, select the best to add limit conditions